docs/content/about/tutorial.md

```diff
@@ -147,64 +151,29 @@ An action is the basic building block in Nikita. It is basically a function, cal
 }
 ```
 
-As you can see, config is made available as the first argument of the handler. This handler is synchronous. Declaring a second argument will mark the function as asynchronous. This second argument is a callback function to be called once the action is done.
-
-```js
-{
-  who: 'leon',
-  handler: function({config}, callback){
-    setImmediate(function(){
-      console.info(config.who)
-    })
-  }
-}
-```
+As you can see, config is made available as the first argument of the handler.
```

Comment:
the handler receive one argument, the action. config is a destructure property of action